variables.tf

```diff
@@ -152,3 +152,9 @@ variable "custom_message" {
   type        = string
   default     = ""
 }
+
+variable "track_as_cluster_level_status" {
+  description = "This allows to check for the status of a cluster in stead of individual hosts, warning and critical thresholds are then expressed as percentages"
```

Comment:
```suggestion
  description = "This allows to check for the status of a cluster instead of individual hosts, warning and critical thresholds are then expressed as percentages"
```